lib/services/nodes-api-service.js

```diff
@@ -280,6 +280,11 @@ function nodeApiServiceFactory(
                     }
                 };
                 return workflowApiService.createAndRunGraph(configuration);
+            } else if (node.type === Constants.NodeTypes.Rack) {
+                return eventsProtocol.publishNodeEvent(node, 'added')
```

Comment:
I think it's fine to add 'added' to other types of nodes events. it's a pre-discovered state, otherwise, it will be a little difficult for documents,  or confuse users in docs.
Forget createNodeAndRunDiscovery(), let me refine it when get a chance :）